app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -116,11 +116,43 @@ public function getFieldAliases($object = 'lead')
         return $qb->select('f.alias, f.is_unique_identifer as is_unique, f.type, f.object')
                 ->from(MAUTIC_TABLE_PREFIX.'lead_fields', 'f')
                 ->where($qb->expr()->eq('object', ':object'))
-                ->setParameter('f.object', $object)
+                ->setParameter('object', $object)
                 ->orderBy('f.field_order', 'ASC')
                 ->execute()->fetchAll();
     }
 
+    /**
+     * Add company left join.
+     *
+     * @param \Doctrine\ORM\QueryBuilder|\Doctrine\DBAL\Query\QueryBuilder $q
+     */
+    public function addCompanyLeftJoin($q)
```

Comment:
I don't see this or getPropertyByField used outside this class. Should they be private?

Reply:
Change to private.